docs/release-notes.rst

```diff
@@ -5,6 +5,15 @@
 Release Notes
 =============
 
+v2024.04.1 (April 1, 2024)
+------------------------------
+This release ...
+
+Documentation
+^^^^^^^^^^^^^
+* Document known issue with `suptitle` argument of  ``set_titles_and_labels`` and Cartopy plots by `Julia Kent`_ in (:pr:`219`)
+* Add alias to ``Taylor_Diagram`` to have more meaningful keyword arguments by `Julia Kent`_ in (:pr:`219`)
```

Comment:
```suggestion

Deprecations
^^^^^^^^^^
* Pending deprecation warnings added for ``TaylorDiagram`` methods ``add_xgrid`` and ``add_ygrid`` which are changing to the new ``add_corr_grid`` and ``add_std_grid`` by `Julia Kent`_ in (:pr:`219`)

```

Reply:
Not committed to this wording / formatting, but I think we should probably have something in here about the deprecations.

Reply:
Oh good point. Slipped my mind trying to make the change quickly. I'll get to this on Monday.